src/targets/gpu/prefuse_ops.cpp

```diff
@@ -182,8 +194,11 @@ struct find_gemm_softmax_gemm
                 .bind("gemm1")));
         auto mul   = match::name("mul")(
             match::nargs(2), match::either_arg(0, 1)(match::is_constant().bind("scale"), gemm1));
+        auto add = match::name("add")(is_bias_supported(),
+                                      match::nargs(2),
+                                      match::any_arg(0, 1)(match::none_of(mul).bind("bias")));
```

Comment:
Is the `bias` term generally a literal or constant ?

Reply:
Hi @umangyadav, 

@manupak asked me to implement the `bias` term as a tensor because it is a more generic approach. So, answering your question, the `bias` is literal